src/vs/base/parts/tree/browser/treeView.ts

```diff
@@ -1556,6 +1558,23 @@ export class TreeView extends HeightMap {
 		return item.top < this.lastRenderTop + this.lastRenderHeight && item.top + item.height > this.lastRenderTop;
 	}
 
+	private parseLabel(label) {
+		let arr = label.split(' ');
+		let columnArr = [];
+		for (let i of arr) {
+			if (i.substring(0, 1) === '(') {
+				break;
+			}
+			else {
+				columnArr.push(i);
+			}
+		}
+		let columnStr = columnArr.join(' ');
+
+		return columnStr;
```

Comment:
I'm assuming this is to handle cases like objects which have `(System-Versioned)` in them. However, this would also get rid of labels that actually have parentheses in them.

Reply:
Are these table fields populated on SQLToolsServer? might be something we can pass in directly.

Also whoever is using parentheses in field/table name really hates their developers 😄